packages/data-context/src/actions/WizardActions.ts

```diff
@@ -313,8 +313,10 @@ export class WizardActions {
       bodyModifier,
     })
 
+    const relativeComponentIndexHtmlPath = path.join('cypress', 'support', 'component-index.html')
+
     return this.scaffoldFile(
-      path.join(this.projectRoot, 'cypress', 'component', 'index.html'),
+      path.join(this.projectRoot, relativeComponentIndexHtmlPath),
```

Comment:
Could just include `this.projectRoot` in the `path.join` on line 316? No need to do `path.join` twice.